java/test/jmri/implementation/AbstractReporterTestBase.java

```diff
@@ -78,6 +78,15 @@ public void testPropertyChange() {
         // Check that LastReport was not seen (no change on null)
         Assert.assertFalse("LastReport seen after null", lastReportSeen);
     }
+    
+    @Test
+    public void testDispose() {
+        Assert.assertEquals("starts 0 listeners", 0, r.getNumPropertyChangeListeners());
+        r.addPropertyChangeListener(new TestReporterListener());
+        Assert.assertEquals("controller listener added", 1, r.getNumPropertyChangeListeners());
+        r.dispose();
+        Assert.assertTrue("controller listeners remaining < 1", r.getNumPropertyChangeListeners() < 1);
```

Comment:
This is still making a call to the object after the dispose call.

Reply:
My suggestion for fixing this is that it is a valid result for a runtime exception to be thrown here.  So but a try catch block around the assert.  If a runtime exception  is thrown, that passes the test.

Reply:
Thanks for fix suggestion, try catch block added.